backend/go.mod

```diff
@@ -151,6 +151,7 @@ require (
 	github.com/mattn/go-isatty v0.0.20 // indirect
 	github.com/mitchellh/copystructure v1.2.0 // indirect
 	github.com/mitchellh/reflectwalk v1.0.2 // indirect
+	github.com/mmcloughlin/avo v0.6.0 // indirect
```

Comment:
why do you need a new dependency?

Reply:
It's probably a new peer / transitive dependency introduced by one of the updates. The indirect comment the Go cli adds, hints at that